x-pack/plugins/cases/server/client/cases/create.ts

```diff
@@ -108,6 +117,15 @@ export const create = async ({
       }),
     });
 
+    // log that we successfully created a case
+    auditLogger?.log(
+      createAuditMsg({
+        operation: Operations.createCase,
+        outcome: EventOutcome.SUCCESS,
+        savedObjectID: newCase.id,
+      })
+    );
+
```

Comment:
I can see that you're logging the outcome of the create operation as well. 
This isn't recommended for the audit log since it creates unnecessary duplication.

Reply:
Thanks, I'll remove it 👍